babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
I added this because the `.babel.rc` file was not working when running `babel-jest` on unit test: Here's where I found the solution https://github.com/facebook/jest/issues/6229#issuecomment-392205646

Reply:
cool, can you please put that comment in the code?  thanks!